### PR TITLE
Add Pastes.io

### PR DESCRIPTION
**Pull request by m7mdharoun**
Add [Pastes.io](https://pastes.io)

### CODE REVIEW

**Comment by lorien** [comment]
Please fix the pull request. At this point github says there is a conflict in merging.

**Comment by m7mdharoun** [comment]
Updated add Pastes.io to the List.

File: README.md [hunk 1]
```diff
@@ -6,6 +6,102 @@ Make a pull request if you want to add new site. I do not accept requests in for
...
+- https://bitbin.it - None
+- https://pastes.io - None
+
+
+## Client Side Encryption
+
+- https://privatebin.net - nothing - 189965
+- https://0bin.net/ - nothing - 235755
+- https://sebsauvage.net/paste/ - nothing - 278090
+- =======
```
Comment (mattreduce) [inline README.md:104]: FYI here's a Git merge conflict marker 👀

File: README.md [hunk 2]
```diff
@@ -94,6 +190,7 @@ Make a pull request if you want to add new site. I do not accept requests in for
 - https://pastebin.fi
 - https://pastething.com
 - https://nekobin.com
+ >>>>>>> main
```
Comment (mattreduce) [inline README.md:193]: Another Git merge conflict marker here

**Pull request closed without merging**